web/ts/components/course.ts

```diff
@@ -137,6 +135,10 @@ export function courseContext(slug: string, year: number, term: string, userId:
             dropdown.toggle(false);
         },
 
+        getMonthName(m: number): string {
+            return Intl.DateTimeFormat("locale", { month: "long" }).format(new Date(m)); // January
```

Comment:
This doesn't work for me, I always get "January" or sometimes the error "date value is not finite in DateTimeFormat.format()"

Maybe we should keep it simple like this, since we don't localize anyway?
```suggestion
            return ["January", "February", "March", "April", "May", "June",
                "July", "August", "September", "October", "November", "December"
            ][m-1];
```

Reply:
Yup, agree.